setup.py

```diff
@@ -47,7 +47,11 @@ def get_version():
     python_requires=">=3.7, <3.12",
     packages=find_packages(),
     install_requires=["numpy>=1.19.0", "gymnasium>=0.26.0", "tinyscaler>=1.2.5"],
-    extras={"dev": ["pettingzoo[butterfly]"]},
+    extras={
```

Comment:
This is just temporary so the CI can be tested on the most recent PZ, will change to 1.23.0 when it is released